### PR TITLE
Create web portfolio

### PR DESCRIPTION
**Pull request by leonard33**
creating my portfolio on web platform

### CODE REVIEW

**Review by Kingstalux — COMMENTED** [review]
Hi @leonard33, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966

## Invalid Code Review Request 🛑 

You have submitted a project, that does not meet the minimum requirements.
- [ ] Kindly update your README file using this [Microverse README template](https://github.com/microverseinc/readme-template) and edit it with your information and information about your project. Make sure all relevant sections are filled before asking for a review 👍 
- [ ] (OPTIONAL) Kindly provide a more detailed pull request description 💯 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -47,15 +47,25 @@
...
+              <li><a href="#">Contact</a></li>
+              <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+          </ul>
+        </div>
       </header>
-      <section>
+      <section class="banner">
+        <div class="introduction">
         <h1 class="name">Hey There. I'm Leonard</h1>
         <h2 class="my-title">I am a Software Developer</h2>
```
Comment (prolajumokeoni) [inline index.html:62]: - [ ] Kindly add the right font color please check the Figma design 👍🏿

File: index.html [hunk 2]
```diff
@@ -47,15 +47,25 @@
...
+          <ul>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+          </ul>
+        </div>
       </header>
-      <section>
+      <section class="banner">
```
Comment (prolajumokeoni) [inline index.html:59]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Template 3-Main  |  web

#### Please note that I reviewed your project at 1312px,  and not 992px the screen size specified in your media query, your project looks more like the design on 1312px and not 992px

- [x] Kindly align the space between the header icons and headline text, to look just like the figma design, please check the screenshot above ⬆️

File: index.html [hunk 3]
```diff
@@ -47,15 +47,25 @@
...
         <a class="owner">John Doe</a>
         <nav class="menu-button"><i class="fa fa-bars"></i></nav>
+        <div class="menu">
+          <ul>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+          </ul>
+        </div>
```
Comment (prolajumokeoni) [inline index.html:57]: - [x] Kindly reduce the height of the header to  look more like the Figma design, please check the screenshot below ⬇️

File: index.html [hunk 4]
```diff
@@ -156,9 +219,12 @@ <h2>Profesional Art Printing Data</h2>
           </ul>
           <button type="button" class="job-button">See Project</button>
         </div>
+      </div>
       </section>
       <section>
```
Comment (prolajumokeoni) [inline index.html:224]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Template 3-Main (2)  | Screenshot_28


- [ ] Kindly align the about me and background image to have the same width with the sublinks 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @leonard33 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly add a more descriptive pull request message explaining changes made in the pull request 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (BenMukebo) [reply index.html:224]: > Figma Design	Your design
> Template 3-Main (2)	Screenshot_28
> * [ ]  Kindly align the about me and background image to have the same width with the sublinks 👍🏿

- [ ] As requested by the previous reviewer, please make sure this paragraph is aligned as per the Figma design
abt

- [ OPTIONAL] In addition to it, Please adjust your styling to match the design of your framework technologies below ⬇️ 
lang

- [ THIS SECTION SHOULD LOOK LIKE THIS INSTEAD]
orgi

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@leonard33,**

You did a great job on making changes requested by the previous reviewer, but there are a few things that are not addressed yet. 👏🏼 👏🏼



### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### ** Hi @leonard33 ** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### - [ ] I strongly recommend you update your readme file in accordance with your project.

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**